File: lista 2 sala/questao 1.py
nota1 = float(input("Digite sua primeira nota:"))
nota2 = float(input("Digite sua segunda nota:"))
nota3 = float(input("Digite sua terceira nota:"))
media = (nota1 + nota2 + nota3) / 3
if media >= 6:
    print(f"Voce foi aprovado! sua nota foi de {media}")
else:
    print(f"Infelizmente voce nao foi aprovado sua media foi de {media}")

File: listas de exercicio/listas 2/lista 2 sala C/questao 1.c
#include <stdio.h>

int main(){
    float N1, N2, N3, M;
    printf("Digite sua primeira nota:");
    scanf("%f", &N1);
    printf("Digite sua segunda nota:");
    scanf("%f", &N2);
    printf("Digite sua terceira nota:");
    scanf("%f", &N3);
    M = (N1 + N2 + N3) / 3;
    if (M >= 6)
    {
        printf("Parabens voce foi aprovado sua media foi de %.2f", M);
    } else{
        printf("Infelizmente voce nao foi aprovado sua media foi de %.2f", M);
    }
    
}